.github/workflows/unittest.yml

```diff
@@ -0,0 +1,32 @@
+---
+name: Run Unit-tests
+
+on: push
+
+jobs:
+  run_unit_test:
+    runs-on: ubuntu-22.04
+    strategy:
+      matrix:
+        python-version: ["3.8", "3.9", "3.10"]
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+          if [ -f dev-requirements.txt ]; then pip install -r dev-requirements.txt; fi
+      - name: Lint with pylinit
+        continue-on-error: true
+        run: |
+          make pylint
```

Comment:
cvprac needs a lot of updates to support pylint on python3 so I'm thinking for now we comment this out?